stl/inc/iterator

```diff
@@ -142,13 +138,14 @@ public:
 #ifdef __cpp_lib_concepts
     using difference_type = ptrdiff_t;
 
-    insert_iterator() = default;
-#else // ^^^ __cpp_lib_concepts / !__cpp_lib_concepts vvv
+    _CONSTEXPR20 insert_iterator(_Container& _Cont, ranges::iterator_t<_Container> _Where)
+        : container(_STD addressof(_Cont)), iter(_STD move(_Where)) {}

```

Comment:
To me, the intent is that we're allowed to move instead of copy everywhere the standard depicts making a copy of an object that meets the _`Cpp17CopyConstructible`_ / _`Cpp17CopyAssignable`_ requirements. _`Cpp17CopyConstructible`_ / _`Cpp17CopyAssignable`_ incorporate _`Cpp17MoveConstructible`_ / _`Cpp17MoveAssignable`_,  so the move operations are required to be valid and produce "equivalent" results.

That said, the difference _is_ observable. Would you like me to submit an LWG issue to add blanket wording permitting an implementation to weaken copies to moves?

Reply:
Yes, I think an LWG issue would be great, thanks. It would need to be carefully phrased since implementations shouldn't be allowed to move from an object multiple times.

In any event, I'm ok with this code as-is.